### PR TITLE
D boost

### PR DESCRIPTION
**Pull request by Quick-Flash**
Tested this for a while now. Seems to fly quite well, haven't heard any complaints with this feature so far. Essentially EmuBoost on the dterm.

### CODE REVIEW

File: src/main/cms/cms_menu_imu.c [hunk 1]
```diff
@@ -126,16 +128,84 @@ static long cmsx_rateProfileIndexOnChange(displayPort_t *displayPort, const void
...
+    return 0;
+}
+
+static OSD_Entry cmsx_menuPidAdvancedEntries[] =
+{
+    { "-- ADVANCED PIDS --", OME_Label, NULL, pidProfileIndexString, 0},
+
+    { "FEATHERED",         OME_UINT8, NULL, &(OSD_UINT8_t){ &feathered_pids,           0, 100, 1}, 0 },
+
+    { "EMU BOOST",         OME_UINT16, NULL, &(OSD_UINT16_t){ &errorBoost,             0,  2000,  5}, 0 },
```
Comment (gretel) [inline src/main/cms/cms_menu_imu.c:181]: the naming scheme is confusing at least. `emu` is superflous.

File: src/main/cms/cms_menu_imu.c [hunk 2]
```diff
@@ -178,10 +243,7 @@ static OSD_Entry cmsx_menuPidEntries[] =
 {
     { "-- EMUFLIGHT PIDS --", OME_Label, NULL, pidProfileIndexString, 0},
 
-    { "FEATHERED", OME_UINT8, NULL, &(OSD_UINT8_t){ &feathered_pids,         0, 100, 1}, 0 },
-
-    { "EMU BOOST", OME_UINT16, NULL, &(OSD_UINT16_t){ &errorBoost,        0,  1000,  5}, 0 },
-    { "BOOST LIMIT", OME_UINT8, NULL, &(OSD_UINT8_t){ &errorBoostLimit,   0,  250,  1}, 0 },
+    {"PID ADVANCED", OME_Submenu, cmsMenuChange, &cmsx_menuPidAdvanced, 0},
```
Comment (gretel) [inline src/main/cms/cms_menu_imu.c:246]: the formatting has been messed up for years..

File: src/main/flight/pid.c [hunk 3]
```diff
@@ -779,6 +784,16 @@ void pidController(const pidProfile_t *pidProfile, const rollAndPitchTrims_t *an
                 dDelta = (float)(kdRingBufferSum[axis] / (float)(pidProfile->dFilter[axis].Wc));
                 kdRingBufferSum[axis] -= kdRingBuffer[axis][kdRingBufferPoint[axis]];
             }
+
+            //dterm boost, similar to emuboost
```
Comment (gretel) [inline src/main/flight/pid.c:788]: emuboost is not explained either in no comments so this refers to nil

**Review by nerdCopter — DISMISSED** [review]
* DBoost feature looks good, does its job.
* l like that it is off by default.
* end-users can use it or not use it, does not break anything.
* Advanced Menu should have been it's own atomic branch, with this adding only DBoost options.

**Comment by nerdCopter** [comment]
* Test and merge #321 first.
* Menu may require cleanup upon updating branch.

**Comment by Quick-Flash** [comment]
ready to be merged.

**Review by nerdCopter — APPROVED** [review]

**Review by nerdCopter — DISMISSED** [review]

**Review by nerdCopter — APPROVED** [review]

**Pull request merged**